features/step_definitions/logging.rb

```diff
@@ -577,15 +577,6 @@
   step %Q/I use the "#{namespace}" project/
 
   unless tagged_upgrade?
-    step %Q/I ensure "#{receiver_name}" service_account is deleted from the "#{namespace}" project after scenario/
-  end
-  @result = user.cli_exec(:create_serviceaccount, serviceaccount_name: receiver_name, n: namespace)
```

Comment:
@anpingli could you reserve the step to create service account? 
Could you please also remove privileged scc from ES deployment files? For ES, just need to update the depoyment files, no need to rebuild image.

Reply:
Once your PR is ready, please remove `[WIP]`

Reply:
@QiaolingTang  Do you mean comment it?

Reply:
no, I was wondering if we could still create the service account but don't add privileged scc to it.

Reply:
@QiaolingTang  The service account is useless if we don't use the privilege pod.  it use the 'default' account by default.

Reply:
PR is not ready yet. I'd like it can be reviewed before drop WIP

Reply:
Ah, I see, thank you.